charts/sn-platform/values.yaml

```diff
@@ -385,6 +385,8 @@ auth:
     # brokerClientCredential: ""
     # brokerClientAuthenticationPlugin: org.apache.pulsar.client.impl.auth.oauth2.AuthenticationOAuth2
     # brokerClientAuthenticationParameters: ""
+    brokerClientCredentialSecret: "my-secret"
```

Comment:
Don't we need to comment these two values out?

Reply:
@tuteng PTAL

Reply:
@fantapsody PTAL